### DISCUSSION TITLE
Targeting .NET 6.0 in Visual Studio 2019 is not supported

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#7025] [General]
**Describe the bug**

Targeting .NET 6.0 in Visual Studio 2019 is not supported

**To Reproduce**

1. Build Avalonia in VS 2019

**Expected behavior**
Do not repoort warnings

**Screenshots**
![immagine](https://user-images.githubusercontent.com/12531229/141965519-118ea37c-7b04-4aea-8592-2f84a7e8f930.png)

**Desktop (please complete the following information):**

- OS: [e.g. Windows, Mac, Linux (State distribution)]
- Version [e.g. 0.10.0-rc1 or 0.9.12]

**Additional context**

**Comment by rstm-sf** [comment]
.NET 6 not supports in VS 2019 https://docs.microsoft.com/en-us/answers/questions/620284/how-to-target-net-60-with-visual-studio-2019.html

https://github.com/dotnet/core/blob/main/release-notes/6.0/known-issues.md#net-sdk

**Comment by maxkatz6** [comment]
We discussed it before, and .NET 6 advantages outweigh the fact you have to install VS2022.

Too bad if Microsoft decided to lock new SDK only for new VS, but it's not something we can deal with.

**Comment by llfab** [comment]
You may want to check this as an intermadiate solution: https://github.com/AvaloniaUI/Avalonia/discussions/7024

**Comment by mysteryx93** [comment]
With JetBrains Rider it supports .NET 6 without a hitch.
